docs/guides/custom-domain.md

```diff
@@ -201,7 +201,7 @@ You could run into connectivity issues if you make a request to your API without
 
 To test connectivity, try the following steps:
 
-1. Deploy any api (e.g. iris-classifier).
+1. Deploy any api (e.g. examples/pytorch/iris-classifier).
```

Comment:
Is this meant to be text-generator?

Reply:
I considered it, it's just that `iris-classifier` is likely to work and is faster to deploy / make an inference, so I decided to go with that one (since the goal is to test networking configuration). What do you think?

Reply:
Makes sense. I agree.